Sources/Bot/Services/GitHubService.swift

```diff
@@ -43,10 +50,37 @@ public final class GitHubService {
             return SignalProducer(error: .unknown)
         }
     }
+
+    private static func signatureVerifier(
+    ) -> (Token) -> (RequestProtocol) -> Result<RequestProtocol, EventHandlingError> {
+
+        return { token in
+            return { request in
```

Comment:
Would be nice to put a comment pointing to the GitHub algorithm description (i.e. [the link you put in the PR body](https://developer.github.com/webhooks/securing/)) somewhere there